test/dynamo/test_activation_checkpointing.py

```diff
@@ -331,6 +331,117 @@ def fn(x, y, z):
         body_function = getattr(cnt.graphs[0], wrap_node.args[0].name)
         self.assertEqual(op_count(body_function), 2)
 
+    @requires_cuda()
+    def test_ddp(self):
+        import os
+
+        import torch.distributed as dist
+        from torch.nn.parallel import DistributedDataParallel as DDP
+        from torch.testing._internal.common_utils import find_free_port
```

Comment:
was it necessary to import these inside the test due to weird import errors from distributed?